kr8s/_auth.py

```diff
@@ -163,6 +163,14 @@ async def _load_kubeconfig(self) -> None:
                     base64.b64decode(self._cluster["certificate-authority-data"])
                 )
                 self.server_ca_file = str(ca_file)
+        if "certificate-authority" in self._cluster:
+            if os.path.isabs(self._cluster["certificate-authority"]):
+                self.server_ca_file = self._cluster["certificate-authority"]
+            else:
+                self.server_ca_file = os.path.join(
+                    os.path.expanduser("~/.kube/"),
```

Comment:
How should this behave if the `KUBE_CONFIG` env var is set? Would it expect to find the certificate authority file in the same directory?

In which case I could recommend using `os.path.dirname(self._kubeconfig)` or similar instead of explicitly using `~/.kube`.

Reply:
Good call! Fixed. 👍